Packs/jamf/ModelingRules/JamfModelingRules/Jamf.yml

```diff
@@ -0,0 +1,18 @@
+name:  Jamf Modeling Rule
+id: jamf_modeling_rule
+fromversion: 6.8.0
+tags: jamf
+rules: ''
+schema: |-
```

Comment:
Shouldn't `groupAddedDevices `  be in the schema as well ?

Reply:
Not sure. This field does not appear in every event type.

Reply:
From what I understood we add the schema that in a case a field doesn't exist it will not fail, I might be wrong but if you're also not sure maybe try to ask in the modeling rules slack channel ?